src/main/java/ToDoList.java

```diff
@@ -0,0 +1,51 @@
+import java.util.ArrayList;
+
+public class ToDoList {
+    private ArrayList<Task> TaskArrayList = new ArrayList<Task>();
+    private Task[] TaskArray = new Task[100];
+    private String dash = "__________________________________";
+
+    private void syncArrayList_Array(){
+        TaskArray = TaskArrayList.toArray(TaskArray);
+    }
+
+    public void addToList(Task taskToBeAdded){
+        TaskArrayList.add(taskToBeAdded);
+        System.out.println(dash+"\nadded: "+taskToBeAdded.name+"\n"+dash);
+        syncArrayList_Array();
+    }
+
+    public void printList(){
+        System.out.println(dash+"\nHere are the tasks in your list:\n");
+        for(int index = 0;index<TaskArrayList.size();index++){
```

Comment:
index could be further expanded on to make it more clear which index it is referring to (better readability)